tests/test_tasks_downstream.py

```diff
@@ -88,12 +106,17 @@ def test_resetdb(
             # Imagine the user is in the odoo subrepo for these tasks
             with local.cwd(tmp_path / "odoo" / "custom" / "src" / "odoo"):
                 # This should install just "base"
-                stdout = invoke("resetdb")
+                stdout = invoke("resetdb", "--no-populate")
             assert "Creating database cache" in stdout
             assert "from template devel" in stdout
             assert _install_status("base") == "installed"
             assert _install_status("purchase") == "uninstalled"
             assert _install_status("sale") == "uninstalled"
+            assert not _get_config_param("report.url")
+            if supported_odoo_version >= 11:
+                stdout = invoke("resetdb", "--populate")  # (default)
```

Comment:
This way you test it's properly the default behavior.

```suggestion
                stdout = invoke("resetdb")  # --populate is the default
```